src/pages/index.vue

```diff
@@ -1,12 +1,16 @@
 <template>
   <div>
-    <v-container fluid>
-      <v-row>
-        <v-col>
-          <v-card color="#BBDEFB">
-            <v-card-title class="headline"> Instructions</v-card-title>
-
-            <v-card-subtitle>
+    <v-container class="py-8">
+      <v-stepper-vertical flat hide-actions editable multiple>
```

Comment:
```suggestion
      <v-stepper-vertical flat hide-actions editable>
```

I prefer to remove the multiple because the UX is a lot clearer. Right now it is also problematic to go back to step one and choose a new file. This will close step 2 that stays opened leaving the user confused, because after uploading, the user expects to see the data table.